### DISCUSSION TITLE
prnounce english words in german model

### DISCUSSION THREAD

**Discussion by pivolan** [coqui-ai/TTS#2850] [General Q&A]
I use vits, a German model trained from scratch on the dataset thorsten, what is the best way to implement correct pronunciation of English words?
You can't add English words to the dataset because it will distort the model due to different phonemicization. 
I see two options, and both are complicated:
1. modify the phonemizer to use the English phonemizer for English words, and thus do the filetuning. And use the same phonemizer for audio generation.
2. use a multi-lingual model, and generate English words separately, and then combine separate audio files, but this will greatly distort the pauses between words. 
Are there any ready solutions for this?

**Comment by erogol** [comment]
your best shot is providing the words with English phonemes.
